python_utils.py

```diff
@@ -431,3 +431,15 @@ def yaml_from_dict(dictionary, width=80):
     """
     dictionary = _recursively_convert_to_str(dictionary)
     return yaml.safe_dump(dictionary, default_flow_style=False, width=width)
+
+def dict_from_yaml(yaml_file):
```

Comment:
I think this should be inside `utils.py`

Reply:
Done.